framework/source/class/qx/ui/treevirtual/TreeVirtual.js

```diff
@@ -155,6 +155,12 @@ qx.Class.define("qx.ui.treevirtual.TreeVirtual",
         };
     }
 
+    if (!custom.tablePaneScroller) {
```

Comment:
Would you please add this to the documentation for `custom` at the top of the constructor.

Reply:
No probs, added

Reply:
Added with latest update

Reply:
Thanks.